validate.lic

```diff
@@ -277,6 +277,46 @@ class DRYamlValidator
             .each { |skill, item| error("Item in weapon_training could not be found in gear listings. '#{skill}: #{item}'") }
   end
 
+  def assert_minimum_whirlwind_trainables(settings)
+    return unless settings.whirlwind_trainables
+    return unless setting.whirlwind_trainables.count < 2
+ 
+    error("Whirlwinding requires a minimum of two one-handed template weapons!")
+  end
+  
+  def assert_that_whirlwind_trainables_are_one_handed(settings)
+    return unless settings.whirlwind_trainables
+    return unless settings.whirlwind_trainables.include?('Twohanded Blunt') || settings.whirlwind_trainables.include?('Twohanded Edged')
+
+    error("Whirlwind does not support twohanded weapons at this time. We only support one-handed weapon templates.")
+  end
+
+  def assert_that_whirlwind_trainables_are_in_weapon_training(settings)
+    return unless settings.weapon_training && settings.whirlwind_trainables
+
+    settings.whirlwind_trainables.each { |trainable|
+      if !settings.weapon_training.has_key?(trainable)
+        error("#{trainable} not found in weapon_training! All whirlwind_trainables need to also be listed in weapon_training.") 
+        return
+      end
+    }
+  end
+
+  def assert_whirlwind_trainables_are_not_swappables(settings)
+    return unless settings.whirlwind_trainables
+    swappables = ['war sword', 'bastard sword', 'throwing spike', 'bar mace', ' half-handled riste', 'riste', 'thick-bladed fan', 'boarding axe', 'broadaxe', 'splitting maul']
+
+    swappables.each { |desc|
+        if settings.weapon_training.has_value?(desc)
+          warn("#{desc} is a swappable! We don't support the swapping of weapon types for whirlwinding. We found this to be too cumbersome and take too much time!") 
```

Comment:
If this will break whirlwinding, I would expect these to be `error`s instead of `warning`s.